.gitattributes

```diff
@@ -20,3 +20,4 @@ phpstan-baseline.neon export-ignore
 phpstan-docs-baseline.neon export-ignore
 phpstan-unsupported-baseline.neon export-ignore
 phpstan.neon.dist export-ignore
+phpunit.xml export-ignore
```

Comment:
I guess we can remove the `phpspec.yml` from line 15

Reply:
Sure, will do it right now.

Reply:
Done!